### DISCUSSION TITLE
How I can to restore my account?

### DISCUSSION THREAD

**Discussion by ykyselv96** [community/community#54726] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello, I have lost all credentials from my old account(https://github.com/ykyselv), because my Ubuntu was broken an all credentials(ssh keys, reсovery code, etc) were stored in my old system. I have some important private and public repos

**Comment by ryanlua** [comment, marked as answer]
Hi @ykyselv96,

It's quite unfortunate to lose your GitHub account. You can reset your password for your GitHub account to regain access to your account by going to "[Reset your password](https://github.com/password_reset)" and entering your email, you should receive an email to the email you used create your GitHub account which will provide the steps to get your password.

If you cannot access your email, you can contact GitHub Support from the "[Help with your GitHub account](https://support.github.com/contact/account)" for additional help.

You can read "[Updating your GitHub access credentials](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/updating-your-github-access-credentials#requesting-a-new-password)" for more information about resetting your password in case you are lost in these steps.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ykyselv96! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ghost** [comment]
how Can I restore the account that I had deleted by mistake
